packages/compass-preferences-model/package.json

```diff
@@ -47,13 +47,12 @@
     "@mongodb-js/compass-logging": "^1.1.6",
     "@mongodb-js/compass-utils": "^0.3.1",
     "ampersand-collection-filterable": "^0.3.0",
-    "ampersand-model": "^8.0.1",
     "ampersand-rest-collection": "^6.0.0",
     "ampersand-state": "5.0.3",
     "bson": "^5.2.0",
+    "joi": "^17.9.2",
     "js-yaml": "^4.1.0",
     "lodash": "^4.17.21",
-    "storage-mixin": "^5.1.5",
```

Comment:
I never realized that this was just not namespaced in any way, not even with the name referencing Ampersand or something like that :)